cursorless-talon/src/check_community_repo.py

```diff
@@ -0,0 +1,34 @@
+from talon import app, registry
+
+required_captures = [
+    "number_small",
+    "user.any_alphanumeric_key",
+    "user.formatters",
+    "user.ordinals_small",
+]
+
+required_actions = [
+    "user.homophones_get",
+    "user.reformat_text",
+]
+
+
+def on_ready():
+    missing_captures = [
+        capture for capture in required_captures if capture not in registry.captures
+    ]
+    missing_actions = [
+        action for action in required_actions if action not in registry.actions
+    ]
+    errors = []
+    if missing_captures:
+        errors.append(f"Missing captures: {', '.join(missing_captures)}")
+    if missing_actions:
+        errors.append(f"Missing actions: {', '.join(missing_actions)}")
+    if errors:
+        errors.insert(0, "https://github.com/talonhub/community")
+        message = "\n".join(errors)
+        app.notify("Cursorless missing community repository", body=message)
```

Comment:
I don't think we want the errors to be in the app notification; I'd put them in a `print` so they go in the log. Most people will just be missing community so they won't care which captures they're missing

Also, is it possible to make it so that if they click the notification it goes to community

Reply:
I have updated the message now. 
I don't think there is any click events on the app notification.